IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_BalanceWeightTensionerDesignCriteria/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_BalanceWeightTensionerDesignCriteria" UniqueId="e42eaebf-5eb5-adab-5725-da15565232bf" ApplicableType="IfcDiscreteAccessory/TENSIONINGEQUIPMENT" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
just double check that these should be on the tensioner itself not one level up in functional grouping.

Reply:
Still think we should keep it.